node/prepublish.js

```diff
@@ -43,15 +43,14 @@ var repoUtils = require('./repoUtils');
 // Use npm.md as the README for the package.
 var absGitRepoPath = path.resolve(path.join(__dirname, '..', '..', '..'));
 var readmePath = path.join(absGitRepoPath, 'README.md');
-var readmeBackupPath = readmePath + '.orig';
+var readmeBackupPath = 'ORIGINAL.md';
```

Comment:
In the `npm publish` case, it's fine, because npm publish sets the current working directory to the package directory, _and_ runs both the pre- and post-publish scripts automatically.  So the pre- and post- scripts will both find the file in the proper location.

In the `npm pack` case, it depends on where you run it from.  If you run the script from the package directory, you'll be fine.  Otherwise...for whatever historical reason, npm pack automatically runs the pre-publish script, but does _not_ run the post-publish step of a package—you have to run that manually.  In this case, like npm publish, the working directory is changed to the package directory, so the pre-publish script will drop `ORIGINAL.md` into that directory.  But if I run the post-publish from somewhere else—and, honestly, I often do, a la

```
$ cd /home/khawkins/npm_test_deploys
$ npm pack /home/khawkins/SalesforceMobileSDK-iOS
$ node /home/khawkins/SalesforceMobileSDK-iOS/external/shared/node/postpublish.js
$ npm install forceios.tgz
```

it's not going to have the right path to move the original file back into place, so the post-publish step will fail.

It's a _fairly_ minor thing.  But I have a special distaste for scripts that are required to run from a specific location. :)